### ISSUE TITLE
Javascript js* form doesn't evaluate parameter in the interpreter before calling javascript eval()

### ISSUE THREAD

**Issue by ktodyruik** [kanaka/mal#150]
I just noticed that js* in the javascript implementation of MAL doesn't evaluate the parameter. It appears to only take string constants.

```
case "js*":
    return eval(a1.toString());
```

What if we wanted to write something like: 

```
(def! alertme (fn* (msg) (js* (str "alert('" msg "')"))))
(alertme "Hi")
```

This code calls javascript eval on this:
```
str,alert(',msg,')
```

**Comment by kanaka** [comment]
Yes, you're right, that's a bug. The way that JavaScript interop works needs to be fixed up (it should work more like miniMAL). I'll see if I can get to that this week (no promises though).

**Comment by kanaka** [comment]
Tracking JS interop fixes in #152
